### DISCUSSION TITLE
Protecting .bcup vault file from deletion

### DISCUSSION THREAD

**Discussion by fraxnet** [buttercup/buttercup-desktop#1234] [General]
The vault file with .bcup extension that was created to save password on local PC can easily be deleted on linux. 
when I tried to protect it with chattr +i file.bcup to prevent deletion, buttercup lost the permission to update/write on the file. 
the question is, how to prevent buttercup vault file from being easily deleted. at least it should ask for confirmation with password to delete it.

**Comment by perry-mitchell** [comment]
This is a good question. I'd strongly advise backing it up regularly - this is the best data loss prevention tactic I can think of. _No protection_ of the file is enough to ensure that it will never be removed/corrupted.

I personally use Nextcloud to sync it.. but many others also use Dropbox, Google Drive, etc.. Syncthing is another option.

Just keeping another copy (and perhaps versioned) is already a great move in protecting such a vital file.
